packages/cli/package/src/lib/dealClient.ts

```diff
@@ -122,6 +131,11 @@ export async function resolveDeployment() {
       return {
         ...DEPLOYMENTS[await ensureChainEnv()],
         ...envConfig?.deployment,
+        // TODO: REMOVE THIS!
```

Comment:
Do we need that?

Reply:
yes for now, soon Valery will fix and I will remove it